Fixtures/Templates/AddTemplate/GoodTemplate/Sources/___NAME___/___NAME___.swift

```diff
@@ -0,0 +1,6 @@
+public struct ___NAME___ {
```

Comment:
We will probably want to use the `___NAME_AS_MODULE___` form here, for cases in which the name isn't already a valid module identifier.

Reply:
Edit: Saw the `___NAME_AS_C99___` further in the code, so that should be used here.